src/e2e/puppeteer/helpers/simulateDragAndDrop.ts

```diff
@@ -0,0 +1,31 @@
+import { Page } from 'puppeteer'
```

Comment:
Let's call this file `dragAndDrop`, since it does not simulate a drag-and-drop, but actually performs a drag-and-drop in the browser.